lib/hermetic-common/src/schema.js

```diff
@@ -249,14 +249,15 @@ const metricSet = Joi.object({
 });
 
 const metric = Joi.object({
-  metricId: Joi.string().required().pk(),
-  name: Joi.string().required().max(70).meta({ displayName: true }),
-  description: Joi.string().required().max(1024),
+  metricId: Joi.string().required().pk().description('Unique identifier for the metric'),
+  name: Joi.string().required().max(70).meta({ displayName: true })
+    .description('The name of the metric which will display in the Portal'),
+  description: Joi.string().required().max(1024).description('A brief summary of what the metric means and how to assess it'),
   category: Joi.string().required()
-    .description('Category to group metrics into'),
+    .description('Free text category which drives how metrics are displayed and aggregated in the Portal'),
```

Comment:
The terminology "the Portal" isn't used in the open source release.  Maybe leave out 'in the Portal'?